workbench_access.module

```diff
@@ -168,9 +168,7 @@ function workbench_access_node_access(NodeInterface $node, $op, AccountInterface
   if ($scheme = $manager->getActiveScheme()) {
     return $scheme->checkEntityAccess($node, $op, $account, $manager);
   }
-  else {
-    return AccessResult::neutral();
-  }
+  return AccessResult::neutral();
```

Comment:
early returns are easier to understand than else statements :heart: - made some similar changes above